CMakeLists.txt

```diff
@@ -91,6 +91,7 @@ endif()
 
 include_directories(BEFORE ${CMAKE_CURRENT_BINARY_DIR}/include/)
 include_directories(BEFORE ${CMAKE_CURRENT_SOURCE_DIR}/include/)
+include_directories(SYSTEM ${CMAKE_CURRENT_SOURCE_DIR}/third_party/)
```

Comment:
also below: can you rebase on master, all access restricted tags should be gone by now - if not please ping me